UIKitTemplate/UIKitTemplate/Views/SelectorCoffeView.swift

```diff
@@ -0,0 +1,274 @@
+        segmentController.frame = CGRect(x: 15, y: 368, width: 345, height: 44)
+        segmentController.setDividerImage(
+            UIImage(named: "segmentControllerColor"),
+            forLeftSegmentState: .selected,
+            rightSegmentState: .normal,
+            barMetrics: .default
+        )
+        for subview in segmentController.subviews {
+            subview.layer.shadowOpacity = 0
+            subview.layer.shadowRadius = 0
+        }
+        return segmentController
+    }()
+
+    /// Надпись "Модификация"
+    private let modificationLabel = {
+        let label = UILabel(frame: CGRect(x: 15, y: 432, width: 200, height: 30))
+        label.text = Constants.modificationTextLabel
+        label.font = .init(name: "Verdana-Bold", size: 18)
+        return label
+    }()
+
+    /// Фон для изображения с кофе
+    private let backgroundCoffe: UIImageView = {
+        let view = UIImageView(frame: CGRect(x: 0, y: 0, width: 375, height: 346))
+        view.image = UIImage(named: "backgroundCoffe")
+        return view
+    }()
+
+    /// Кнопка "Заказать"
+    private let makeOrderButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 15, y: 717, width: 345, height: 53))
+        button.setTitle(Constants.makeOrderButtonLabel, for: .normal)
+        button.backgroundColor = UIColor(named: "orderButtonColor")
+        button.titleLabel?.font = .init(name: "Verdana-Bold", size: 18)
+        button.layer.cornerRadius = 12
+        return button
+    }()
+
+    /// Вью с кофе
+    private var coffeView: UIImageView = {
+        let view = UIImageView(frame: CGRect(x: 112, y: 128, width: 150, height: 150))
+        view.image = UIImage(named: "americano")
+        return view
+    }()
+
+    /// Кнопка выбора обжарки кофе
+    private lazy var roastSelectionButton = CustomUIButton().createButton(
+        withImageName: "darkRost",
+        labelText: Constants.darkOfRoastButtonText,
+        position: CGPoint(x: 15, y: 482)
+    )
+
+    /// Кнопка дополнительных ингредиентов
+    private lazy var aditivesButton = CustomUIButton().createButton(
+        withImageName: "plusImage",
+        labelText: Constants.aditivesButtonLabel,
+        position: CGPoint(x: 195, y: 482)
+    )
+
+    // MARK: - Public Properties
+
+    /// Ссылка на делегата
+    weak var delegate: SelectorCoffeDelegate?
+
+    // MARK: - Initializers
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        createView()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Public Methods
+
+    /// Метод обновления вью в зависимости от выбранного кофе
+    /// - Parameters:
+    ///   - withState: выбранное состояние селект контроллера
+    func reloadView(withState state: Int) {
+        switch state {
+        case 0:
+            coffeView.image = UIImage(named: "americano")
```

Comment:
```suggestion
            coffeView.image = UIImage(named: dictionary[stateIndex])
```
через словарь в 1 строку сделай